### ISSUE TITLE
Failed to run `cargo build --release` on Termux

### ISSUE THREAD

**Issue by ytest6709** [uutils/coreutils#5817]
`cargo run --release` failed to install
When I tried, it got error
```
~/coreutils $ cargo build --release
   Compiling rustix v0.37.26
   Compiling syn v1.0.109
   Compiling strsim v0.10.0                                                      Compiling blake3 v1.5.0
   Compiling arrayref v0.3.6
   Compiling constant_time_eq v0.3.0
   Compiling thiserror v1.0.37
   Compiling time-core v0.1.0
   Compiling data-encoding v2.5.0
   Compiling arrayvec v0.7.4
   Compiling same-file v1.0.6
   Compiling either v1.8.0
   Compiling walkdir v2.4.0
   Compiling terminal_size v0.2.6
   Compiling time-macros v0.2.8
   Compiling clap_builder v4.4.2
   Compiling keccak v0.1.4
   Compiling nix v0.27.1
   Compiling num_threads v0.1.6
   Compiling once_cell v1.19.0
   Compiling data-encoding-macro-internal v0.1.12
   Compiling thiserror-impl v1.0.37
   Compiling uuhelp_parser v0.0.23 (/data/data/com.termux/files/home/coreutils/src/uuhelp_parser)
   Compiling clap v4.4.2
   Compiling itoa v1.0.4
   Compiling time v0.3.20
   Compiling data-encoding-macro v0.1.14
   Compiling uucore_procs v0.0.23 (/data/data/com.termux/files/home/coreutils/src/uucore_procs)
   Compiling sha3 v0.10.8
   Compiling itertools v0.12.0
   Compiling blake2b_simd v1.0.2
   Compiling sha1 v0.10.6
   Compiling sha2 v0.10.8
   Compiling os_display v0.1.3
   Compiling sm3 v0.4.2
   Compiling md-5 v0.10.6
   Compiling glob v0.3.1
   Compiling wild v2.2.0
   Compiling dunce v1.0.4
   Compiling z85 v3.0.5
   Compiling hex v0.4.3
   Compiling autocfg v1.1.0
   Compiling uucore v0.0.23 (/data/data/com.termux/files/home/coreutils/src/uucore)
   Compiling num-traits v0.2.17
   Compiling aho-corasick v1.0.4
error[E0277]: the trait bound `i64: From<u64>` is not satisfied
   --> src/uucore/src/lib/features/fsext.rs:672:29
    |
672 |         return self.f_bsize.into();
    |                             ^^^^ the trait `From<u64>` is not implemented for `i64`
    |
    = help: the following other types implement trait `From<T>`:
              <i64 as From<bool>>
              <i64 as From<i8>>
              <i64 as From<i16>>
              <i64 as From<i32>>
              <i64 as From<u8>>
              <i64 as From<u16>>
              <i64 as From<u32>>
              <i64 as From<NonZeroI64>>
    = note: required for `u64` to implement `Into<i64>`

error[E0277]: the trait bound `i64: From<u64>` is not satisfied
   --> src/uucore/src/lib/features/fsext.rs:750:28
    |
750 |         return self.f_type.into();
    |                            ^^^^ the trait `From<u64>` is not implemented for `i64`
    |
    = help: the following other types implement trait `From<T>`:
              <i64 as From<bool>>
              <i64 as From<i8>>
              <i64 as From<i16>>
              <i64 as From<i32>>
              <i64 as From<u8>>
              <i64 as From<u16>>
              <i64 as From<u32>>
              <i64 as From<NonZeroI64>>
    = note: required for `u64` to implement `Into<i64>`

For more information about this error, try `rustc --explain E0277`.
error: could not compile `uucore` (lib) due to 2 previous errors
warning: build failed, waiting for other jobs to finish...
```
Please fix this now.

**Comment by sylvestre** [comment]
> Please fix this now.

please avoid such comments.

**Comment by BenWiederhake** [comment]
This was apparently fixed in #5939, and even changes how Termux is tested in CI: 93d922f07593dbf3fceeb03ea5d53f5169820e7a
